### PR TITLE
Issue #176 cal license notification

### PR DESCRIPTION
**Pull request by jagriffin5**
Add check to Analysis Summary for CAL licenses usage and when the number of schedulers are less than the number of CPUs.  This is an info warning and would still require engineer to validate but brings light to the low scheduler use along with the CAL license which restricts to 20 physical or 40 logical cores.

### CODE REVIEW

File: sqlnexus/PerfStatsAnalysis.sql [hunk 1]
```diff
@@ -3478,6 +3485,46 @@ END
...
+		FROM [tbl_ServerProperties] WHERE PropertyName in ('number of visible schedulers')
+	END
+
+	IF @cpuCount <> 0 OR @cpuCount is not null
+	BEGIN
+		IF EXISTS (SELECT 1 FROM [dbo].[tbl_ServerProperties] where PropertyName = 'Edition' AND PropertyValue NOT LIKE '%Core-based%') AND @cpuCount > @schedCount 
+			BEGIN
+				UPDATE [dbo].[tbl_AnalysisSummary]
+				SET [Status] = 1,
+				[Description] = 'Customer is using CAL license and CPUs (' + CONVERT(VARCHAR, @cpuCount) + ') are greater than schedulers (' + CONVERT(VARCHAR, @schedCount) + ') online, check the errorlog to confirm. Customer could benefit by upgrading to CORE license.'
```
Comment (PiJoCoder) [inline sqlnexus/PerfStatsAnalysis.sql:3521]: @jagriffin5 , keep in mind that this is a public product and audience is everyone. So changing the verbiage a bit. If you like the proposed verbiage, you can just a accept/commit the suggestion.

```suggestion
				[Description] = 'Using CAL license and CPUs (' + CONVERT(VARCHAR, @cpuCount) + ') are greater than schedulers (' + CONVERT(VARCHAR, @schedCount) + ') online, check the errorlog to confirm. The user is not taking full advantage of available hardware and can benefit from upgrading to CORE license.'
```

**Comment by PiJoCoder** [comment]
@jagriffin5 I see you have create a PR. Are you ready for us to review or are you still working on it? I also just made a suggestion for a change

**Comment by jagriffin5** [comment]
Yes, it is good to go.  I noticed the comments and approved/committed those suggestions.  Thank you for that.

**Comment by PiJoCoder** [comment]
> Yes, it is good to go. I noticed the comments and approved/committed those suggestions. Thank you for that.

Great! I will test and review within the next 2-3 days and merge if all looks good.

**Pull request merged**